commands/General/Role Manipulation/manageroles.js

```diff
@@ -0,0 +1,86 @@
+const { Command } = require('klasa');
+const { RichDisplay } = require('klasa');
+const { MessageEmbed } = require('discord.js');
+
+/* You need to add the following lines to your guild schematic manager:
+     	.add('selfroles', folder => folder
+	    .add('roles', 'role', { array: true })
+            .add('enabled', 'boolean', { default: true }))
+*/
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			runIn: ['text'],
+			requiredPermissions: ['SEND_MESSAGES', 'EMBED_LINKS', 'USE_EXTERNAL_EMOJIS', 'MANAGE_ROLES'],
```

Comment:
`SEND_MESSAGE` is generic and already checked in the monitor, so not needed.
`USE_EXTERNAL_EMOJIS`? Is that _really_ needed?